cmd/namespace/delete.go

```diff
@@ -53,6 +53,10 @@ func executeDeleteNamespace(ctx context.Context, namespace string) error {
 		return fmt.Errorf("failed to delete namespace: %s", err)
 	}
 
+	if err := CleanNamespace(ctx, namespace); err != nil {
```

Comment:
nit: I'd make this lowercase, since it should be private to this package only

Reply:
Feels like it should mirror RunNamespace, which is sorta its opposite. Thoughts?

Reply:
Ah, good point.